mne/io/eeglab/eeglab.py

```diff
@@ -347,8 +347,8 @@ def __init__(self, input_fname, montage, eog=(), event_id=None,
         info['chs'].append(stim_chan)
         info._update_redundant()
 
-        events = _read_eeglab_events(eeg, event_id=event_id,
-                                     event_id_func=event_id_func)
+        events = read_events_eeglab(eeg, event_id=event_id,
+                                    event_id_func=event_id_func)
```

Comment:
I don't see how that makes sense. The problem is the following:
- EEGLAB events can be arbitrary strs
- MNE stim chans or mne.Epochs(..., events, ...) must be int

The user *knows* what events are in the data, and in case they've forgotten, we print the str events in the warning. What we have to do is, we have to allow users to map strs to ints. So in this one, the workflow goes like this:

```
raw = mne.io.read_raw_eeglab(fname)
-> Warning: Events like the following will be dropped entirely:
"square", "circle". 100 event codes could not be mapped to integers.
Use the 'event_id' parameter to map such events manually.

event_id = {"square":1, "circle":2}
raw = mne.io.read_raw_eeglab(fname, event_id)
events = mne.find_events(raw)
epochs = mne.Epochs(raw, events, event_id)
for cond in event_id:
    epochs[cond].average().plot(title=cond)
```

Seems fine to me. Or am I missing what you're going for?